app/views/admin/pages/_form.html.erb

```diff
@@ -10,95 +10,112 @@
+        <%= form.text_field :title, class: "crayons-textfield", required: true %>
+      </div>
+      <div class="crayons-field">
+        <%= form.label :slug, class: "crayons-field__label" %>
+        <%= form.text_field :slug, class: "crayons-textfield", pattern: "^[0-9a-z\-_]*$", required: true %>
+      </div>
+      <div class="crayons-field">
+        <%= form.label :description, class: "crayons-field__label" %>
+        <%= form.text_field :description, class: "crayons-textfield", required: true %>
+      </div>
+      <div class="crayons-field">
+        <%= form.label :template, class: "crayons-field__label" do %>
+          Template
+          <p class="crayons-field__description">Determines the way page's body will be embedded in the layout</p>
+        <% end %>
+        <%= form.select :template, Page::TEMPLATE_OPTIONS, required: true %>
+      </div>
+      <div class="crayons-field optional html">
+        <%= form.label :body_markdown, class: "crayons-field__label" %>
+        <%= form.text_area :body_markdown, rows: 10, class: "crayons-textfield" %>
+      </div>
+      <div class="crayons-field optional html">
+        <%= form.label :body_html, class: "crayons-field__label" do %>
+          Body HTML
+          <p class="crayons-field__description">Only if not using markdown. HTML is dangerous ⚠️</p>
+        <% end %>
+        <%= form.text_area :body_html, rows: 10, class: "crayons-textfield" %>
+      </div>
+      <div class="crayons-field optional json">
+        <%= form.label :body_json, class: "crayons-field__label" do %>
+          Body JSON
+          <p class="crayons-field__description">For use with template type <code>json</code></p>
+        <% end %>
+        <%= form.text_area :body_json, rows: 10, class: "crayons-textfield" %>
+      </div>
+      <div class="crayons-field">
+        <% if @page.social_image_url %>
+          <img src="<%= @page.social_image_url %>" style="max-width:500px;display:block" />
+        <% end %>
+        <%= form.label :social_image, class: "crayons-field__label" %>
+        <%= form.file_field :social_image %>
+      </div>
+      <div class="crayons-field crayons-field--checkbox">
+        <%= form.check_box :is_top_level_path, class: "crayons-checkbox" %>
+        <%= form.label :is_top_level_path, class: "crayons-field__label" do %>
+          Is top level path?
+          <p class="crayons-field__description">Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code> Be careful! ⚠️</p>
+        <% end %>
+      </div>
+
+      <% if ForemInstance.private? %>
+        <div class="crayons-field crayons-field--checkbox">
+          <%= form.label :landing_page, class: "crayons-field__label" do %>
+            Use as 'Locked Screen'
+            <p id="lock-screen-description" class="crayons-field__description">Determines if this page will be used as a landing page for anonymous viewers.</p>
```

Comment:
I was just looking at the E2E failures and noticed one of them is because this changes the label to be 

` "Use as 'Locked Screen' Determines if this page will be used as a landing page for anonymous viewers."`

The `<p>` should be outside of the `form.label` so that it can act just as a description rather than part of the label name. I already approved but we should defo give this one a quick fix!

Reply:
I wanted to keep that `p` inside label so i changed the test approach a little bit - nothing really drastic hopefully.

Reply:
It's not great for screen reader users to have such a lengthy label name 😕 

But if we really do want it to be part of the label, we should remove the `"aria-describedby": "lock-screen-description"` on line 72 because it's recycling the label text as the description now 🙃